src/Datadog.Trace/Vendors/StatsdClient/StatsdUDP.cs

```diff
@@ -47,13 +47,13 @@ public StatsdUDP(string name = null, int port = 0, int maxUDPPacketSize = Statsd
 
         private static string GetHostNameFromEnvVar()
         {
-            return Environment.GetEnvironmentVariable(StatsdConfig.DD_AGENT_HOST_ENV_VAR);
+            return EnvironmentHelpers.GetEnvironmentVariable(StatsdConfig.DD_AGENT_HOST_ENV_VAR);
```

Comment:
See [this comment](https://github.com/DataDog/dd-trace-dotnet/pull/747/files#r435456373).